### ISSUE TITLE
PMM crashes only when analyzing series

### ISSUE THREAD

**Issue by ShanaryS** [Kometa-Team/Kometa#1319]
### Version Number

1.18.3

### What branch are you on?

master

### Describe the Bug

Once PMM starts processing the first show, it freezes for a minute then the container exists. This is only for analyzing series or anime series. It works perfectly for movies and anime movies.

### Logs

https://gist.github.com/ShanaryS/10048b52d6b979560a482336c3d4d967

**Comment by YozoraXCII** [comment]
Can you run with `--trace` at the end of your Run Command and provide those logs?

Generally speaking, if PMM exits iself due to an issue, you will see an error message and the normal end-run summary. This *seems* like something else is killing it.

I see that your host only has 2GB of RAM, are you able to increase that and see if it helps?

**Comment by ShanaryS** [comment]
I used the env variable PMM_TRACE=True so it was on in the logs.

The RAM was the issue. I'm using docker on windows and WSL2 had it's ram limited to 2GB. I upped it to 16GB (32GB system) and it now works. RAM went to 3GB where it froze above and went as high as 9.1GB later on when applying overlays. This is the WSL2 total memory, docker strangely says the specific container only uses 2.5GB but nothing else is running.

Might be a good idea to add a note about RAM requirements in the setup so others can avoid this in the future.
